README.md

```diff
@@ -42,15 +42,15 @@ Benefits
 
 - Requires minimal skills to get started (Just HTML)
 - Creates accessible prototypes
-- Once learned,then it provides Experience of much faster than any drag-and-drop tool.
-- Create dynamic designs that flex & change depending on the width of the screen
+- Once learned,it provides Experience a much faster prototyping experience than visual drag-and-drop tool.
```

Comment:
This sentence is still awkward. I recommend changing it to "Once learned, it provides a much faster prototyping experience than visual drag-and-drop tools."

Reply:
Okay sir , Just changing it in some minutes.